roles/env_vars/templates/env.j2

```diff
@@ -59,15 +50,9 @@ DJANGO_SLSX_VERIFY_SSL_INTERNAL={{ slsx_verify_ssl_internal }}
 DJANGO_SLSX_VERIFY_SSL_EXTERNAL={{ slsx_verify_ssl_external }}
 
 # Misc. settings
-DJANGO_BLOCK_HTTP_REDIRECT_URIS="{{ django_block_http_redirect }}"
-DJANGO_OAUTH2_PROVIDER="{{ django_oauth2_provider }}"
 DJANGO_LOGGING="{{ django_logging }}"
-DJANGO_NO_UI="{{ django_no_ui }}"
 CORS_ORIGIN_ALLOW_ALL="{{ django_cors_origin_allow_all }}"
 
-# Tealim config
-{% if env in ["prod", "impl"] %}TEALIUM_ENV="prod"{% endif %}
-
```

Comment:
Each removed variable here was carefully grepped for throughout the bluebutton-web-server repo to ensure no reliance.  The variables were either not used at all, or set by default and never pulled from the .env file.